services/worker-manager/schemas/v1/register-worker-response.yml

```diff
@@ -36,7 +36,18 @@ properties:
     required:
       - accessToken
       - clientId
```

Comment:
Add `workerConfig` here.  It's good that worker-runner is resilient to the field not being present, but we can let consumers written after this lands just assume it's present (and catch bugs where it's omitted).